tests/integration/components/content-test.js

```diff
@@ -0,0 +1,68 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import { assertTooltipContent } from '../../helpers/ember-tooltips';
+
+moduleForComponent('tooltip-on-element', 'Integration | Option | content', {
+  integration: true,
+});
+
+test('assertTooltipContent correctly matches expected tootltip content', function(assert) {
+
+  assert.expect(2);
+
+  this.render(hbs`
+    <div>
+      :)
+      {{tooltip-on-element text='Smiley face'}}
+    </div>
+  `);
+
+  assertTooltipContent(assert, {
+    contentString: 'Smiley face',
+  });
+
+  this.render(hbs`
+    <div>
+      :(
+      {{#tooltip-on-element}}
+        Frowning face
+      {{/tooltip-on-element}}
+    </div>
+  `);
+
+  assertTooltipContent(assert, {
+    contentString: 'Frowning face',
+  });
+});
+
+test('assertTooltipContent correctly compares expected and discovered tooltip content of tooltip', function(assert) {
+
+  assert.expect(2);
+
+  this.render(hbs`
+    <div>
+      :)
+      {{tooltip-on-element text='Smiley face'}}
+    </div>
+  `);
+
+  const stubbedAssert = {
```

Comment:
This just stubs the `equal` on `assert.equal` that is passed in to our helper. The value this adds is testing that our helper calls what we think it will with the values it should have.